.github/workflows/ci.yml

```diff
@@ -32,8 +32,11 @@ jobs:
       run: printenv
       shell: bash
 
+    - name: 'Upgrade pip'
+      run: python -m pip install --upgrade pip
+
     - name: 'Install dependencies'
-      run: python -m pip install --upgrade -r requirements.txt
+      run: pip install -r requirements.txt
```

Comment:
Why did you revert the removal of the (duplicated) requirements of the first commit? Doesn't the documented `pip install -e .[full]` work?